tests/unit/test_rag_server/test_rag_main_core_components.py

```
@@ -77,22 +82,32 @@ def test_init_with_valid_vdb_op(self):
 
     def test_init_with_invalid_vdb_op(self):
         """Test initialization with invalid vdb_op type."""
-        with pytest.raises(ValueError, match="vdb_op must be an instance of nvidia_rag.utils.vdb.vdb_base.VDBRag"):
+        with pytest.raises(
+            ValueError,
+            match="vdb_op must be an instance of nvidia_rag.utils.vdb.vdb_base.VDBRag",
```

Ruff (0.14.8)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)